### PR TITLE
Port over Derived Table Instance Alternative Titles to metadb

### PR DESCRIPTION
**Pull request by jenmawe**
This is a derived table for the instance_alternative_titles converted from LDP for Metadb. I kept the same naming conventions and removed the folio_derived schema. 

There is no issue related to this pull request.

### CODE REVIEW

File: sql_metadb/derived_tables/instance_alternative_titles.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS instance_alternative_titles;
+
+CREATE TABLE instance_alternative_titles AS 
+SELECT 
+	i.id AS instance_id,
+	jsonb_extract_path_text(i."jsonb", 'hrid') AS instance_hrid,
+	jsonb_extract_path_text(titles.jsonb, 'alternativeTitle') AS alternative_title,
+    jsonb_extract_path_text(titles.jsonb, 'alternativeTitleTypeId')::uuid AS alternative_title_type_id,
+    titles.ORDINALITY AS alternative_title_ordinality,
```
Comment (nassibnassar) [inline sql_metadb/derived_tables/instance_alternative_titles.sql:9]: Here `ordinality` is the name of a column (I know it is confusing) and so it should be all in lowercase.

File: sql_metadb/derived_tables/instance_alternative_titles.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
+DROP TABLE IF EXISTS instance_alternative_titles;
+
+CREATE TABLE instance_alternative_titles AS 
+SELECT 
+	i.id AS instance_id,
+	jsonb_extract_path_text(i."jsonb", 'hrid') AS instance_hrid,
```
[the remark this answers is not in the document]
Comment (jenmawe) [reply sql_metadb/derived_tables/instance_alternative_titles.sql:6]: DBeaver is doing this automatically. The quotes are probably in my other ones here up for review. I'll remove them in the others I submit.

File: sql_metadb/derived_tables/instance_alternative_titles.sql (hunk 1 above)
Comment (jenmawe) [reply sql_metadb/derived_tables/instance_alternative_titles.sql:9]: DBeaver automatically puts this in uppercase. I can change it to lowercase.

**Review by adoerrer — CHANGES_REQUESTED** [review]
Query looks good and results are as aspected. Only have some minor changes requested. Thank you!

File: sql/report_queries/HathiTrust reports/Serials Holdings Report/README.md [hunk 3]
```diff
@@ -8,7 +8,13 @@ This report generates data with the following format:
 
 | title | instance\_id | instance\_hrid | oclc\_number | instance\_mode\_of\_issuance | item\_stat\_code\_type\_name | item\_stat\_code\_name | hol\_stat\_code\_type\_name | hol\_stat\_code\_name | inst\_stat\_code\_type\_name | inst\_stat\_code\_name | location\_name | campus\_name | library\_name | institution\_name |
 |---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
+<<<<<<< HEAD
+| The year FOLIO went live | 0b96a642-5e7f-452d-9cae-9cee66c9a892 | item000000000017 | 098766 | serial | ARL (Collection stats) | Serials, print (serials) | ARL (Collection stats) | Serials, print (serials) | ARL (Collection stats) | Serials, print (serials) | Main Library | City Campus | FOLIO Libraries | |
+
+
+=======
 | The year FOLIO went live | 0b96a642-5e7f-452d-9cae-9cee66c9a892 | item000000000017 | 098766 | text | Available | 69640328-788e-43fc-9c3c-af39e243f3b7 | ARL (Collection stats) | Serials, print (serials) | ARL (Collection stats) | Serials, print (serials) | ARL (Collection stats) | Serials, print (serials) | Stacks | Main Campus | Library | Institution |
+>>>>>>> d2ecb37f8f3ac73f0d599ec2bb9c0a2cb5bfd538
```
[the remark this answers is not in the document]
Comment (nassibnassar) [reply sql/report_queries/HathiTrust reports/Serials Holdings Report/README.md:17]: Good catch.  The lines that begin with `<<<<<<<`, `=======`, and `>>>>>>>` look like they are leftover from a merge with a conflict that was supposed to be resolved manually.

File: sql_metadb/derived_tables/instance_alternative_titles.sql (hunk 2 above)
[the remark this answers is not in the document]
Comment (nassibnassar) [reply sql_metadb/derived_tables/instance_alternative_titles.sql:6]: Yes, sorry.  There is definitely a way to configure that in the DBeaver settings, but it is not obvious where to look.  This is probably something we should figure out and document somewhere.

File: sql_metadb/derived_tables/instance_alternative_titles.sql [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+CREATE INDEX ON instance_alternative_titles (instance_hrid);
+
+CREATE INDEX ON instance_alternative_titles (alternative_title);
+
+CREATE INDEX ON instance_alternative_titles (alternative_title_type_id);
+
+CREATE INDEX ON instance_alternative_titles (alternative_title_ordinality);
+
+CREATE INDEX ON instance_alternative_titles (alternative_title_type_name);
```
[the remark this answers is not in the document]
Comment (nassibnassar) [reply sql_metadb/derived_tables/instance_alternative_titles.sql:26]: Yes, sorry, I added those probably around the same time these commits were created.

**Comment by nassibnassar** [comment]
Closing at Jennifer Eustis' request.

**Pull request closed without merging**